Description: Do not install the useless .la file.
 This is a FreeBSD-specific patch, upstream doesn't really need it.
Author: Ying-Chieh Liao <ijliao@FreeBSD.org>
Forwarded: not-needed
Last-Update: 2009-09-10

--- ltmain.sh.orig
+++ ltmain.sh
@@ -6003,10 +6003,12 @@
 	fi
 
 	# Install the pseudo-library for information purposes.
+	if /usr/bin/false; then
 	name=`$echo "X$file" | $Xsed -e 's%^.*/%%'`
 	instname="$dir/$name"i
 	$show "$install_prog $instname $destdir/$name"
 	$run eval "$install_prog $instname $destdir/$name" || exit $?
+	fi
 
 	# Maybe install the static library, too.
 	test -n "$old_library" && staticlibs="$staticlibs $dir/$old_library"
